float32 target_current
---
bool state